src/CMakeLists.txt

```diff
@@ -1128,20 +1128,17 @@ if(USE_ABC)
   target_include_directories(cvc4 PRIVATE ${ABC_INCLUDE_DIR})
 endif()
 if(USE_CADICAL)
-  target_link_libraries(cvc4 ${CaDiCaL_LIBRARIES})
-  target_include_directories(cvc4 PRIVATE ${CaDiCaL_INCLUDE_DIR})
+  target_link_libraries(cvc4 CaDiCaL)
```

Comment:
Link PRIVATE?

Reply:
This needs to be tone for all calls at once... see #6218.